### PR TITLE
Added Item package to represent Items and changed Character.java accordingly. 

### PR DESCRIPTION
**Pull request by edwardsues**
Not completely finished, still waiting for InventoryManager implementation and will work on equip/unequip methods in character later.

### CODE REVIEW

File: src/main/java/Character.java [hunk 1]
```diff
@@ -31,4 +41,26 @@ public boolean removeItem(String item){
...
 
+    public int getCurrentHealth(){
+        return this.currentHealth;
+    }
+
+    public int getMaxHealth() {
+        return maxHealth;
+    }
+
+    public void setCurrentHealth(int currentHealth) {
```
Comment (wBernie) [inline src/main/java/Character.java:52]: maybe have an adjust currenthealth method as well. (adding or subtracting a value vs. setting it to a value)

File: src/main/java/items/ConsumableStatusItem.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        super(name, description);
+        this.statusType = statusType;
+    }
+
+    public String getStatusType(){
+        return this.statusType;
+    }
+
+    public void consume(Character consumer){
+        if (this.statusType == consumer.getStatus){
```
Comment (wBernie) [inline src/main/java/items/ConsumableStatusItem.java:21]: what if you're affected by multiple status types

**Review by georgewesley — APPROVED** [review]
Read over everything and it looks good!

File: src/main/java/Character.java (hunk 1 above)
Comment (edwardsues) [reply src/main/java/Character.java:52]: I was thinking we leave the logic in combat/consumable: e.g. we do the calculation and then just call the method of setting it.

File: src/main/java/items/ConsumableStatusItem.java (hunk 2 above)
Comment (edwardsues) [reply src/main/java/items/ConsumableStatusItem.java:21]: honestly, not sure, i think we can just leave it at one status type at atime

Comment (wBernie) [reply src/main/java/items/ConsumableStatusItem.java:21]: in that case, how do we want to handle status application on someone already affected by a different one. Will it not work or will it replace the old one.

**Review by wBernie — APPROVED** [review]

**Pull request merged**